.github/workflows/main.yml

```diff
@@ -44,7 +44,6 @@ jobs:
                    "mssql", "mssql17",
                    "mongo", "mongo4",
                    "firestore",
-                   "airtable",
```

Comment:
Why did you delete this?

Reply:
We never really supported this, all the tests are running on some mock implementation that we maintain, so in every new test I need to add more complexity to the mock or not support this test.
Anyway, I talked with @michaelir, and we decided to let it go for now.